src/js/index.js

```diff
@@ -0,0 +1,62 @@
+import homePage from './page-load';
+import menu from './menu';
+import navbar from './navbar';
+import contact from './contact';
+
+require('../css/style.css');
+
```

Comment:
### Use ES6 syntax
- [ ] Use import for CSS files and remember to have a configuration for CSS files loader in webpack file